package.json.sh

```diff
@@ -25,43 +25,24 @@ browser:
 
 files:
     'bin/latex.js'
-
-    'dist/index.js'
```

Comment:
You removed `index.js` but you didn't update `main:` further up in the file.

Reply:
I think we need to get `index.js` back. Otherwise, plain node users will download the latexjs package with lots of dependencies in `node_modules`, but none of them would be used, and none could be updated, because we just have this one bundle.